bin/setup_venv.sh

```diff
@@ -235,16 +235,34 @@ else
   exit 1
 fi
 
-# Add local kubectl to the virtual environment
+#
+# This section originally pulled the most recent version of Kubectl down; however it turned out that
+# was causing isues with our AWS deploy (see the issues in the repo). Addtionally, this was only
+# downloading the kubectl if it did not exist; this could result in versions not being updated if the
+# MARA project was run in the same environment w/o a refresh.
+#
+# The two fixes here are to hardcode (For now) to a known good version (1.23.6) and force the script to
+# always download this version.
```

Comment:
Anytime that we are hardcoding a version into a non-standard place I want to make sure that we explain why. My thought is that anyone who starts digging into the setup_venv code will understand this.

Ideally, we will set this value at the same place as everything else; I tried to put it into requirements.txt and that failed with errors (pip tries to install it, which is weird because it doesn't try and install other things we download like pulumi).

I'm going to open up an issue to make sure contributors see this.